client/me/purchases/cancel-purchase/cancellation-effect.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import { getName, getSubscriptionEndDate, isRefundable } from 'lib/purchases';
+import { isDomainMapping, isGoogleApps, isJetpackPlan, isTheme } from 'lib/products-values';
+
+function cancellationEffectHeadline( purchase, translate ) {
+	const { domain } = purchase;
+	const	purchaseName = getName( purchase );
+
+	if ( isRefundable( purchase ) ) {
+		return translate(
+			'Are you sure you want to cancel and remove %(purchaseName)s from {{em}}%(domain)s{{/em}}? ', {
+				args: {
+					purchaseName,
+					domain
+				},
+				components: {
+					em: <em />
+				}
+			}
+		);
+	}
+
+	return translate(
+		'Are you sure you want to cancel %(purchaseName)s for {{em}}%(domain)s{{/em}}? ', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Are you sure you want to cancel and remove %(purchaseName)s from {{em}}%(domain)s{{/em}}? ' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).